Modules/Numerics/FEM/include/itkImageToRectilinearFEMObjectFilter.h

```diff
@@ -57,7 +57,7 @@ class ITK_TEMPLATE_EXPORT ImageToRectilinearFEMObjectFilter : public ProcessObje
   /** Run-time type information (and related methods). */
   itkTypeMacro(ImageToRectilinearFEMObjectFilter, ProcessObject);
 
-  static constexpr unsigned int NDimensions = TInputImage::ImageDimension;
+  static constexpr unsigned int VDimension = TInputImage::ImageDimension;
```

Comment:
I recommend against this edit.  Instead I would edit https://github.com/InsightSoftwareConsortium/ITK/blob/114dcc15f2fbf7b1bb3b33049f067fec5674586c/Modules/Numerics/FEM/include/itkImageToRectilinearFEMObjectFilter.h#L72
to use `NDimensions`, because there is the possibility that modules external to ITK make use the publicly declared class member of `ImageToRectilinearFEMObjectFilter`.

Quite possibly all occurrences of `FEMObject<VDimension>` will need to be changed to use `NDimensions` in
```
Modules/Numerics/FEM/include/itkImageToRectilinearFEMObjectFilter.h
Modules/Numerics/FEM/include/itkFEMSolver.h
Modules/Numerics/FEM/include/itkMetaFEMObjectConverter.hxx
Modules/Numerics/FEM/include/itkFEMObject.hxx
```

Reply:
No, I think I am wrong about the additional occurrences.  Hopefully just the one edit to https://github.com/InsightSoftwareConsortium/ITK/blob/114dcc15f2fbf7b1bb3b33049f067fec5674586c/Modules/Numerics/FEM/include/itkImageToRectilinearFEMObjectFilter.h#L72 will be sufficient.

Reply:
Instead of that file list above, I am seeing errors at only the following.  We would change `VDimension` back to `NDimensions` in these places.

1. Modules/Numerics/FEM/include/itkImageToRectilinearFEMObjectFilter.h:72
2. Modules/Numerics/FEM/include/itkImageToRectilinearFEMObjectFilter.hxx:35
3. Modules/Numerics/FEM/include/itkImageToRectilinearFEMObjectFilter.hxx:140

Reply:
And

4. Modules/Numerics/FEM/include/itkImageToRectilinearFEMObjectFilter.hxx:37